src/static_canvas.class.js

```diff
@@ -1117,14 +1104,11 @@
      * @private
      * @param {fabric.Object} object Object to center
      * @param {fabric.Point} center Center point
-     * @return {fabric.Canvas} thisArg
-     * @chainable
      */
     _centerObject: function(object, center) {
       object.setPositionByOrigin(center, 'center', 'center');
       object.setCoords();
       this.renderOnAddRemove && this.requestRenderAll();
-      return this;
```

Comment:
i would like to limit breaking changes in 5. Those are not part of the animation changes. What is the reason to remove chainability now?

Reply:
It's as you said. I thought it's too weird to have the animated version not chainable while the regular one is.